src/Member.java

```diff
@@ -0,0 +1,12 @@
+public interface Member {
+    void run();
+    void jump();
+    String getName();
+    boolean isSuccess();
+    void setSuccess(boolean success);
+    double getMaxRunDistance();
+    double getMaxJumpHeight();
+}
```

Comment:
no, we don't want to have this number of methods in the interface.  Need to be split into smaller interfaces. Take a closer look at SOLID and particularly to "I" in it.  Also, Member does not sound like an action and interface is a contract meeting that object can do some action